src/js/components/Marathon.jsx

```diff
@@ -94,7 +97,9 @@ var Marathon = React.createClass({
       view = decodeURIComponent(view);
     }
 
-    var activeTabId = tabs[0].id;
+    var activeTabId = !this.isActive("404")
```

Comment:
You could use ```this.context.router.isActive("404")``` to be consistent with the other places where we are using the context and get rid of the RouterStateMixin in this component.